Sources/ForageSDK/ForageSDK.swift

```diff
@@ -12,8 +12,8 @@ public class ForageSDK {
     private static var config: Config?
     static var logger: ForageLogger?
     var service: ForageService?
-    var merchantID: String = ""
-    var sessionToken: String = ""
+    var merchantID: String = config?.merchantID ?? ""
+    var sessionToken: String = config?.sessionToken ?? ""
```

Comment:
ty @dleis612 for proposing this!

Reply:
There is the downside of this approach that these values are randomly the empty string, which may be hard for us to debug in the future. I think we should just put this down as tech debt to be solved later.